docs/meetings/2018-03-18-JENKINS-49406-quality-bar/README.adoc

```diff
@@ -0,0 +1,69 @@
+= [2018-03-18] How to test the data snapshotting system (JENKINS-49406)
+
+We wanted to discuss testing link:https://issues.jenkins-ci.org/browse/JENKINS-49406[https://issues.jenkins-ci.org/browse/JENKINS-49406] itself, but also possibly the  overall quality bar footnote:[the different aspects of the software we make sure to test, generally trying to define them *before* the software is actually developed :-)] for Jenkins Essentials.
```

Comment:
Defined before and that need to be met before going to production

Reply:
I would also call it "quality requirements " instead of "aspects of the software"

Reply:
@vilacides fixed, WDYT?